pkg/domain/workflow.go

```diff
@@ -39,3 +44,9 @@ type WorkflowListener struct {
 	URL          string
 	DashboardURL string
 }
+
+// AssignedCodeset describes an assigned codeset its webhook ID
```

Comment:
Missing _and_ between _codeset_ and _its_?

Reply:
done